README.md

```diff
@@ -179,6 +179,9 @@ jobs:
         with:
           manifest_file: example-app/spin.toml
           fermyon_token: ${{ secrets.FERMYON_CLOUD_TOKEN }}
+          key_values: |-
```

Comment:
Would a more natural YAML be:

```
key_values:
  abc: xyz
  foo: bar
```

We don't need to bring the CLI syntax across verbatim...

Reply:
this is a Github limitation. Inputs to actions does not support that syntax and the workaround is to pass as follows:

```
          key_values: |-
            abc=xyz
            foo=bar
```

Reply:
Wow, that's disappointing.  I guess they are trying to keep the data types minimal so that inputs can always fit into environment variables.